docs/guides/device-gateway/sdk.md

```diff
@@ -0,0 +1,344 @@
+---
+sidebar_position: 9
+title: Connect to Devices Using the Python SDK
+description: Connect to devices or your device gateway by embedding connectivity directly into your application with the Python SDK.
+tags:
+  - guides
+  - devices
+  - gateway
+  - security
+  - fleet
+  - iot
+  - sdk
+  - python
+---
+
+This guide provides step-by-step instructions for using ngrok as a device gateway.
+This example shows you how to embed connectivity into an application using ngrok's Python SDK to access
+an API running on an IoT device. All the Python code in this example is available in
+[this repo](https://github.com/ngrok-samples/ngrok-python-iot-agent).
+
+## Prerequisites
+
+To use the sample application, you need the following prerequisites on the device or device
+gateway you wish to connect to:
+
+- Python 3.11
+- [venv](https://docs.python.org/3/library/venv.html#) or another Python virtual environment manager
+
+## Get an ngrok API Key
+
+[Create an ngrok API key](https://dashboard.ngrok.com/api/new) using the ngrok dashboard.
+Make sure you save the API key before you leave the screen because it won't be displayed again.
+
+## Create a custom wildcard domain
+
+Next, create a custom [wildcard domain](/docs/api/resources/reserved-domains/), which will allow you to
+create endpoints and receive traffic on any subdomain of your domain.
+
+For example, you might create `*.sitea.{YOUR_DOMAIN}`. You would then be able to create endpoints
+on `device342.sitea.{YOUR_DOMAIN}` and `device896.sitea.{YOUR_DOMAIN}` without having to reserve them individually.
+It can be helpful to create a separate subdomain for each site you wish to connect to.
+
+Run the following command, substituting your API key for `{API_KEY}` and your domain for `{YOUR_DOMAIN}`:
```

Comment:
@DevMandy @stmcallister (and honestly everyone else who writes docs...)

We should come up with a standard way of defining these "variables" in code blocks that users need to find and replace. I don't have a strong opinion about which one, but I think if we're consistent internally then they'll more easily be able to pick them out of complex code/config.

Any thoughts?

Reply:
Completely agree! I ❤️ consistency! I would vote to find out if we already use a particular standard more often and stick to that one. I don't have strong feelings about which style of syntax we use just that it's consistent.

Reply:
At the moment, these are consistent across guides (or at least new guides). I can look at making them consistent as part of the guides reboot project. I agree they should be consistent.

Reply:
I was using `<API_KEY>` in previous guides, but I'm happy to follow your path here with the brackets. They do stick out nicely from typical YAML/JSON syntax.